frontend/public/components/_resource.scss

```diff
@@ -11,13 +11,18 @@
   padding: 2px 4px 1px;
   text-align: center;
 
-  &--lg {
-    font-size: ($font-size-base + 1);
-    line-height: 21px;
-    margin-right: 7px;
-    min-width: 24px;
-    padding: 0 7px;
-  }
+}
+
+.co-m-resource-icon--lg {
```

Comment:
IMO it's better to for elements and modifiers nested so there is no need to duplicate the block.

Reply:
Ok I nested this and the ones related to `co-resource-item`